examples/escrow/1.0.0.json

```diff
@@ -117,10 +130,11 @@
         "address": "0x35b6b723786fd8bd955b70db794a1f1df56e852f",
         "transaction": "0x905fbbeb6069d8b3c8067d233f58b0196b43da7a20b839f3da41f69c87da2037",
         "block": "0x9b39dcab3d665a51755dedef56e7c858702f5817ce926a0cd8ff3081c5159b7f",
-        "link_dependencies": [
-          {"offset": 524, "value": "SafeSendLib"},
-          {"offset": 824, "value": "SafeSendLib"}
-        ]
+        "runtime_bytecode": {
+          "link_dependencies": [
+            {"offsets": [262, 412], "value": "SafeSendLib"}
```

Comment:
Some brief thoughts I'm having now about the `value` field.  Currently, we use three formats which are all distinctly distinguishable from each other

- `ContractName`
  - unique because it isn't one of the other two formats.
- `package-a:package-b:ContractName`
  - unique because of the embedded `:`
- `0xdeadbeef`
  - unique because it starts with a `0`

Should we just define this as it's own object with a `type` field

- `{'type': 'reference', 'value': 'SafeMathLib}`
- `{'type': 'literal', 'value': '0xdeadbeef'}`

My gut says yes because this pattern of regex based string differentiation has bit me before, and I *suspect* that it will again.  This specific case doesn't seem to have any edge cases which result in ambiguity, but it would be **very** easy for one to get introduced by accident without us realizing it.

Thoughts?

Reply:
I agree with going with an object for the `value` field as opposed to regex based string differentiation 👍🏻objects are more open for extension as well as easier to parse for people new to a codebase imo

Reply:
Seem like we can just elevate it to the main object

```javascript
{"offsets": [262, 412], "type": "reference", "value": "SafeSendLib"}
```